core/crypto/nike/nike.go

```diff
@@ -46,9 +52,12 @@ type PublicKey interface {
 	Blind(blindingFactor []byte) error
 }
 
-// Nike is an interface encapsulating a
+// Scheme is an interface encapsulating a
 // non-interactive key exchange.
-type Nike interface {
+type Scheme interface {
```

Comment:
Scheme seems awfully generic and hard to search for, `NikeScheme` maybe?

Reply:
It's in the "nike" module thus making the API stutter: nike.NikeScheme.
We're following circl's example here: https://github.com/cloudflare/circl/blob/main/kem/kem.go#L33